### PR TITLE
Update about-telerik-reporting.md

### PR DESCRIPTION
**Pull request by todorarabadzhiev**

### CODE REVIEW

File: embedding-reports/display-reports-in-applications/web-application/native-blazor-report-viewer/how-to-use-native-blazor-report-viewer.md [hunk 1]
```diff
@@ -1,40 +1,40 @@
 ---
 title: How to Use Native Blazor Report Viewer
-page_title: How to Use Native Blazor Report Viewer 
-description: How to Use Native Blazor Report Viewer
+page_title: Integrating Native Blazor Report Viewer in Blazor Server or WebAssembly Application
+description: "Learn how to configure and integrate the Native Blazor Report Viewer built on top of Telerik Blazor UI widgets in you Blazor Server or WebAssembly Application."
```
Comment (bankovaprgs) [inline embedding-reports/display-reports-in-applications/web-application/native-blazor-report-viewer/how-to-use-native-blazor-report-viewer.md:4]: I think the correct term is Telerik UI for Blazor https://www.telerik.com/blazor-ui

File: embedding-reports/display-reports-in-applications/web-application/native-blazor-report-viewer/send-mail-message.md [hunk 2]
```diff
@@ -1,26 +1,26 @@
 ---
 title: Send Mail Message
-page_title: Send Mail Message
-description: Send Mail Message
+page_title: Send Mail Message Configuration
+description: "Learn how to enable and the Send Mail Message button of the Native Blazor Report Viewer and how to pre-configure the Send Mail Message Dialog."
```
Comment (bankovaprgs) [inline embedding-reports/display-reports-in-applications/web-application/native-blazor-report-viewer/send-mail-message.md:4]: the first and is not needed 
Learn how to enable the Send

File: getting-started/about-telerik-reporting.md [hunk 3]
```diff
@@ -47,19 +47,17 @@ The report definition is the model that contains the declarations of all the rep
 
 To create a report, you need to generate a report definition file by using the supported [Telerik Report Designers]({%slug telerikreporting/designing-reports/report-designer-tools/overview%}).
 
-* If you design a report in the Report Designer for Visual Studio, the result is a class inheriting `Telerik.Reporting.Report`.
+* If you design a report in the Report Designer for Visual Studio, the result is a class inheriting `Telerik.Reporting.Report`. These are the so called __Type Report Definitions__. They may be referenced also as _CLR_ (Common Language Runtime), _CS_ (C Sharp) or _VB_ (Visual Basic) report definitions. These separate CLR types may be passed to the Reporting engine through the built-in [TypeReportSource](/api/telerik.reporting.typereportsource).
```
Comment (bankovaprgs) [inline getting-started/about-telerik-reporting.md:50]: Great, thank you Toshe. The link is returning 404, but I guess it will be ok on test and live docs

**Review by bankovaprgs — APPROVED** [review]

File: getting-started/about-telerik-reporting.md (hunk 3 above)
Comment (todorarabadzhiev) [reply getting-started/about-telerik-reporting.md:50]: yes, it will be OK, the API ref is built with a separate process

**Pull request merged**